pkg/crc/preflight/preflight_linux.go

```diff
@@ -140,6 +140,10 @@ func checkVsock() error {
 }
 
 func fixVsock() error {
+	// first check if crc daemon is running
```

Comment:
Nit: Since the next line uses a helper with a nice name (`checkIfCRCDaemonRunning`), this comment is redundant, code reading is smoother if you remove it